packages/suite/src/views/wallet/trading/common/TradingForm/TradingBuyFormInputs.tsx

```diff
@@ -101,7 +101,7 @@ export const TradingBuyFormInputs = () => {
                         />
                     </Column>
 
-                    {cryptoId && !isLoading && <TradingReceiveAddress type="buy" />}
+                    {cryptoId && !isLoading && <TradingReceiveAddress />}
```

Comment:
This is intentional. The type isn't needed since the divider has been removed from there.